locales/en-US/webextension.properties

```diff
@@ -27,6 +27,8 @@ loginErrorDetails = We couldn’t save your shot because there is a problem with
 unshootablePageErrorTitle = We can’t screenshot this page.
 unshootablePageErrorDetails = This isn’t a standard Web page, so you can’t take a screenshot of it.
 selfScreenshotErrorTitle = You can’t take a shot of a Firefox Screenshots page!
+# Fired when someone makes a zero-width or zero-height selection
+emptySelectionErrorTitle = Your selection is too small
```

Comment:
@flodolo We're aiming for 55, but have a new string to add. Is it possible to get strings uplifted after 55 graduates to Beta?